### DISCUSSION TITLE
在win11里添加的开始菜单里的磁贴无法应用到win10开始菜单里

### DISCUSSION THREAD

**Discussion by chenpaner** [zetaloop/ExplorerPatcher#15] [General]
用软件切换开始菜单风格为win10后，原先在win11里添加的开始菜单里的磁贴无法应用到win10开始菜单里，而且无法手动添加（或者说是不知道怎么重新添加磁贴）！
在个人的开始菜单文件里有些快捷键无法显示在win10的磁贴里，但可以显示在win11的磁贴里，
我的的开始菜单文件：C:\Users\CP\AppData\Roaming\Microsoft\Windows\Start Menu\Programs

**Comment by DEAN-Cherry** [comment]
https://github.com/valinet/ExplorerPatcher/discussions/1085
